test/Dommel.IntegrationTests/Databases/SqlServerDatabaseDriver.cs

```diff
@@ -9,9 +9,16 @@ public class SqlServerDatabaseDriver : DatabaseDriver
 {
     public override DbConnection GetConnection(string databaseName)
     {
-        var connectionString = CI.IsAppVeyor
-            ? $"Server=(local)\\SQL2017;Database={databaseName};User ID=sa;Password=Password12!"
-            : $"Server=(LocalDb)\\mssqllocaldb;Database={databaseName};User ID=dommel;Password=dommel";
+        var connectionString = $"Data Source=.;Initial Catalog={databaseName};Integrated Security=True;TrustServerCertificate=True";
```

Comment:
Does Integrated Security work for all databases in the Docker compose setup?

Reply:
In works on my local machine.